src/main/java/org/truffleruby/parser/TranslatorDriver.java

```diff
@@ -151,7 +151,8 @@ public RubyRootNode parse(RubySource rubySource, ParserContext parserContext, St
                 context,
                 isInlineSource,
                 !isEvalParse,
-                false);
+                false,
+                context.getCoreLibrary().isLoadingRubyCore());
```

Comment:
This feels a bit fragile as it relies on global state.
Maybe we should use BodyTranslator#inCore instead (that should probably be cached tough), and use that in BodyTranslator#visitDStrNode.
Then ParserConfiguration wouldn't have to know about this.

It might be useful to know if in core or not in ParserConfiguration in general though, I'm not sure.

I think fine to merge as-is, we can reconsider it later.

Reply:
> Maybe we should use BodyTranslator#inCore instead

So set the property in the parser node, then ignore it during translation?

Reply:
Yes, that's what I was thinking.